test/gurka/gurka.h

```diff
@@ -818,6 +818,37 @@ void expect_maneuvers(const valhalla::Api& result,
       << "Actual maneuvers didn't match expected maneuvers";
 }
 
+/**
+ * Tests whether the expected set of instructions is emitted for the specified maneuver index.
+ * Looks at the output of Odin in the result.
+ *
+ * @param result the result of a /route or /match request
+ * @param maneuver_index the specified maneuver index to inspect
+ * @param expected_instructions the set of four instructions expected in the DirectionsLeg for the
+ *                              route at specified maneuver index.
+ *                              The four instructions shall be in this order:
```

Comment:
Nit: how about using an unordered_map instead of a vector so that order doesn't matter?

Reply:
but then the function call would have more clutter with it - looks nice and simple now:
``` cpp
  gurka::assert::raw::expect_instructions_at_maneuver_index(
      result, maneuver_index,
      {"Enter Dupont Circle and take the 1st exit toward A 95/B 2/München/Kürten.",
       "Enter Dupont Circle and take the 1st exit toward A 95.",
       "Enter Dupont Circle and take the 1st exit toward A 95, München.", ""});
```

Reply:
The downside of using a vector is that the caller needs to know/remember the order in which the items need to be put. Using a hashmap makes it a bit more explicit:
```c++
gurka::assert::raw::expect_instructions_at_maneuver_index(
    result, maneuver_index,
      {{"text", "Enter Dupont Circle and take the 1st exit toward A 95/B 2/München/Kürten."},
       {"verbal_transition", "Enter Dupont Circle and take the 1st exit toward A 95."},
       {"verbal_pre_transition", "Enter Dupont Circle and take the 1st exit toward A 95, München."},
       {"verbal_post_transition", ""}});
```
But I leave it to you. I guess its okay with vector too. We'll learn the order :)

Reply:
Let's discuss Monday - we can add an issue to switch it if needed